MonoGame.Framework/MonoGame.Framework.WindowsDX.csproj

```diff
@@ -78,6 +80,8 @@
     <PackageReference Version="4.0.1" Include="SharpDX.MediaFoundation" />
     <PackageReference Version="4.0.1" Include="SharpDX.XAudio2" />
     <PackageReference Version="4.0.1" Include="SharpDX.XInput" />
+    <PackageReference Include="System.Net.Http" Version="4.3.4" />
+    <PackageReference Include="System.Security.Cryptography.X509Certificates" Version="4.3.2" />
```

Comment:
Why are these and NETCore.Jit / .NETCore.Aop packages needed?

Reply:
See my note in the original PR statement

> https://github.com/MonoGame/MonoGame/pull/8089/commits/5b02edfb4ebf339440dbe37b44fe66d852c33bd4 `MonoGame.Framwork.WindowsDX` Transient packages from `SharpDX`, `SharpDXDirect2D1`, `SharpDX.Direct3D11`, `SharpDX.DXGI`, `SharpDX.MediaFoundation`, `SharpDX.XAudio2`, and `SharpDX.XInput` are marked as high risk security vulnerability.  Tried updating SharpDX directly, but new version (4.2.0) has breaking changes to the `IAsyncCallback` interface.  Opted to instead update the transient dependencies directly as top-level pacakges:
>    - `Microsoft.NETCore.App` updated to 2.1.30
>    - `Microsoft.NETCore.Jit` updated to 2.0.8
>    - `System.Net.Http` updated to 4.3.4
>    - `System.Security.Cryptography.X509Certificates` updated to 4.3.2

tl;dr, these packages were marked as high risk vulnerability, but updating the main packages directly created breaking changes due to **SharpDX** introducing `IAsyncCallback`, so the transient packages with the vulnerabilities were updated instead.